queries/c/highlights.scm

```diff
@@ -177,7 +180,11 @@
   declarator: (identifier) @parameter)
 
 (parameter_declaration
-  declarator: (pointer_declarator) @parameter)
+  declarator: (pointer_declarator) @pointer.declarator (#set! "priority" 101))
+
```

Comment:
make the pointer parameter(name)  color with `@parameter` hl group priority to overcome the pointer declarator..